docs/faq.rst

```diff
@@ -24,6 +26,36 @@ comparing `jinja2`_ to `Django`_.
 .. _jinja2: http://jinja.pocoo.org/
 .. _Django: https://www.djangoproject.com/
 
+How can I use Scrapy with BeautifulSoup?
+----------------------------------------
+
+As mentioned :ref:`above <faq-scrapy-bs-cmp>`, BeautifulSoup can be used
+for parsing HTML responses in Scrapy callbacks.
+You just have to feed the response's body into a ``BeautifulSoup`` object
+and extract whatever data you need from it.
+
+Here's an example spider using ``lxml`` parser with BeautifulSoup API::
```

Comment:
Just test, what you get without "lxml" is:

```
2016-06-10 19:17:34 [scrapy] DEBUG: Crawled (200) <GET http://www.example.com/> (referer: None)
2016-06-10 19:17:34 [py.warnings] WARNING: /home/paul/.virtualenvs/scrapybs4.py3/lib/python3.5/site-packages/bs4/__init__.py:166: UserWarning: No parser was explicitly specified, so I'm using the best available HTML parser for this system ("lxml"). This usually isn't a problem, but if you run this code on another system, or in a different virtual environment, it may use a different parser and behave differently.

To get rid of this warning, change this:

 BeautifulSoup([your markup])

to this:

 BeautifulSoup([your markup], "lxml")

  markup_type=markup_type))

2016-06-10 19:17:35 [scrapy] DEBUG: Scraped from <200 http://www.example.com/>
{'url': 'http://www.example.com/', 'title': 'Example Domain'}
```
so it works but it's not the cleanest output.
but I agree we can keep the example minimal, and maybe add a in-code comment about adding "lxml"